scripts/util.js

```diff
@@ -20,8 +20,41 @@ util.articleTab = function () {
 
   });
 };
+util.filterViewByAuthor = function () {
 
+  var source = $('#author-filter-template').html();
+  var template = Handlebars.compile(source);
+  var compiledHtml = template(blog);
+  $('.author-filter-attach').append(compiledHtml);
+
+  $('.author-filter').change(function() {
+    $('article').show();
+    $('.category-filter').children().removeAttr('selected');
+    var $selectedAuthor = $('.author-filter option:selected').val();
+    var slugSelected = blog.slugify($selectedAuthor);
+    $('span:not(:contains('+ slugSelected +'))').parent().hide();
+
+  });
+};
+
+util.filterViewByCategory = function () {
+  var source = $('#category-filter-template').html();
+  var template = Handlebars.compile(source);
+  var compiledHtml = template(blog);
+  $('.category-filter-attach').html(compiledHtml);
+
+  $('.category-filter').change(function(){
+    $('article').show();
+    $('.author-filter').children().removeAttr('selected');
+    var selectedCategory = $('.category-filter option:selected').val();
+    $('article h6:not(:contains('+ selectedCategory+'))').parent().hide();
+
+  });
+
```

Comment:
this is fine but consider moving these to blog object rather than util (think of util as a place for helpers like slugify that don't care so much about content). both these filter methods contains a lot of information specific to blog, so they fit best in the blog object (as blog methods).